Makefile

```diff
@@ -1127,3 +1165,53 @@ $(GOLANGCI_LINT): .github/workflows/golangci-lint.yml # Download golangci-lint u
 	hack/ensure-golangci-lint.sh \
 		-b $(TOOLS_BIN_DIR) \
 		$(shell cat .github/workflows/golangci-lint.yml | grep [[:space:]]version | sed 's/.*version: //')
+
+$(GINKGO): # Build ginkgo from tools folder.
+	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) $(GINKGO_PKG) $(GINKGO_BIN) $(GINGKO_VER)
+
+
+## --------------------------------------
+## Templates
```

Comment:
Q: should we keep those near generate-e2e-templates? (no strong opinions here)

Reply:
We have them here and there to separate different parts. I kind of like them, but I also have no strong opinions here.